src/common/io.cc

```diff
@@ -335,6 +335,10 @@ void read_message(int fd, int64_t *type, int64_t *length, uint8_t **bytes) {
 
 disconnected:
   /* Handle the case in which the socket is closed. */
+  // RAY_LOG(INFO) << "Socket Disconnected in read_message "
+  //               << static_cast<int64_t>(MessageType::DisconnectClient);
+  RAY_LOG(INFO) << "Socket Disconnected in read_message "
```

Comment:
this codepath isnt hit

Reply:
Right, this is a legacy Ray thing which is replaced by `client_connection.cc` I believe